### PR TITLE
Address a handful of warnings from #6095

### PR DESCRIPTION
**Pull request by ZeldaZach**
Solved a handful of the warnings, but there are several we can't control with protobuf. Should turn those off within the protobuf headers.

### CODE REVIEW

File: cockatrice/src/tab_deck_editor.cpp [hunk 1]
```diff
@@ -1248,7 +1249,7 @@ void TabDeckEditor::showSearchSyntaxHelp()
                .replace(QRegularExpression("^(##)(.*)", opts), "<h2>\\2</h2>")
                .replace(QRegularExpression("^(#)(.*)", opts), "<h1>\\2</h1>")
                .replace(QRegularExpression("^------*", opts), "<hr />")
-               .replace(QRegularExpression("\\[([^\[]+)\\]\\(([^\\)]+)\\)", opts), "<a href=\'\\2\'>\\1</a>");
+               .replace(QRegularExpression(R"(\[([^[]+)\]\(([^\)]+)\))", opts), R"(<a href='\2'>\1</a>)");
```
Comment (ebbit1q) [inline cockatrice/src/tab_deck_editor.cpp:1252]: I don't think this is a readability improvement when all the others use the escaped backslashes

Comment (ZeldaZach) [reply cockatrice/src/tab_deck_editor.cpp:1252]: This wasn't readable to begin with, but the linter recommended I change it. It no longer has the incorrect escape sequence, which was lost in all the backslashes to begin with.

Comment (ebbit1q) [reply cockatrice/src/tab_deck_editor.cpp:1252]: you mean the escaped [?

Comment (ZeldaZach) [reply cockatrice/src/tab_deck_editor.cpp:1252]: Yeah, which wasn't necessary

**Review by ebbit1q — APPROVED** [review]
looks fine

**Pull request merged**